tools/scripts/build_projects.py

```diff
@@ -38,17 +38,28 @@ def parse_input():
 		args.platform, args.build_name)
 
 ERR = 0
+LOG_START = " -> "
```

Comment:
do we need this ? If there is a an error will they also get the " -> " sign ?

Don't have a strong opinion on it. We can go with it as is.

Reply:
yes, -> will be also on error. 
This will help indentation following the change in color of -> will make easier to find the error.